core/src/test/java/io/questdb/griffin/CopyTest.java

```diff
@@ -155,196 +152,202 @@ public void testCopyFullHack2() throws Exception {
 
     @Test
     public void testSimpleCopy() throws Exception {
-        assertMemoryLeak(() -> {
-
+        ParallelCopyRunnable insert = () -> {
```

Comment:
Probably we should rename `ParallelCopyRunnable` into `CopyRunnable`. Same with `testParallelCopy`. They no longer stand for parallel COPY only.